Works Section of Portfolio Project (Mobile Version)

engrabdullahriaz opened the pull request:
1. Added all the requirements for colors, fonts, and box-model from the Figma design
2. No linters error
3. README is descriptive
4. created a new branch

engrabdullahriaz committed:
Updated README with my social network links

engrabdullahriaz committed:
added file required for linters

engrabdullahriaz committed:
node_modules in gitignore

engrabdullahriaz committed:
Added background header image of figma design

engrabdullahriaz committed:
portfolio html file

engrabdullahriaz committed:
portfolio css file

engrabdullahriaz committed:
html & css linters file

engrabdullahriaz committed:
Updated css file font family

engrabdullahriaz committed:
Changed css file for font family

engrabdullahriaz committed:
updated css file

engrabdullahriaz committed:
updated css file

engrabdullahriaz committed:
updated html file for hamburger

engrabdullahriaz committed:
Designed html file for mobile version

engrabdullahriaz committed:
updated css file

engrabdullahriaz committed:
adding resource images

engrabdullahriaz committed:
Updated css file and fixed linter error

engrabdullahriaz committed:
updated html file for mobile version

vic778 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
```
kindly the social media should be flexed. You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

vic778 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+      </ul>
+    </div>
+
+    </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/Portfolio.png" alt="">
+        </div>
+        <div class="card-header">
+          <h3>Multi-post <br> stories</h3>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Frontend Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2022</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">HTML</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">CSS</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">Javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
```
kindly the `h3` enter in the `see project` fix that issue

vic778 replied:
Kindly this section should display the image of each projects. and the project should have the see project button according to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

vic778 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+          </div>
+          <div class="about-language">
+            <div class="lang-title">
+              <h2>Languages</h2>
```
Kindly this title miss an arrow down icon and try to reduce the title is too big differently to the Figma template

engrabdullahriaz replied:
<div class="lang-title">
              <h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>
            </div>

engrabdullahriaz replied:
<h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>

vic778 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+            <div class="skills-language">
+              <h3>Framework</h3>
+              <ul>
+                <li class="arrow-right"><i class="fa fa-arrow-right"></i></li>
```
Kindly this font-awesome is not working according to the Figma template

vic778 commented on index.html:
```diff
@@ -0,0 +1,136 @@
+              <div class="skills-info">
+              <h4 class="skills-title">skills</h4>
+              <ul>  
+              <li class="arrow-side"><i class="fa fa-arrow-right"></i></li>
```
Kindly this font-awesome is not working according to the Figma template

vic778 reviewed (changes requested):
## Hello @engrabdullahriaz 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
-  Good video description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

engrabdullahriaz committed:
updated html file

engrabdullahriaz committed:
updated html file

engrabdullahriaz committed:
updated html file

engrabdullahriaz committed:
updated html file

engrabdullahriaz committed:
updated html file

AnselemOdims commented on index.html:
```diff
@@ -0,0 +1,136 @@
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <h3><button class="btn-card">See project</button></h3>
```
- [ ] As pointed out by the previous reviewer, please make sure that each card has an image like the original design.
- [ ] Also, please ensure to include the number of cards in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164946303-39bd6a96-b740-49ad-8806-28d12b14f5d7.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164946351-e386185a-f5b0-428e-b517-124a1adf3c29.png)

AnselemOdims commented on index.html:
```diff
@@ -0,0 +1,136 @@
+      <!--end of work section-->
+
+      <!--end about me section-->
+      <section class="about_me">
+        <div class="about_container">
+          <h3 class="about_me_title">About<br>Myself</h3>
+          <div class="about_info">
+            <h3 class="about_me_title-desktop">About me</h3>
+            <div class="about_description">
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don’t
+                hestiate to contact me.
+              </p>
+            </div>
+            <div>
+              <button type="button" class="about_btn">Get My Resume</button>
+            </div>
+          </div>
+          <div class="about-language">
+            <div class="lang-title">
+              <h3>Languages</h3>
+              <ul>
+              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
```
- [ ] Please kindly ensure the feedback by the previous reviewer are implemented and the about section looks like the original design

![image](https://user-images.githubusercontent.com/43995830/164946439-700f792e-15bd-4f26-8c21-d81c88033f6b.png)

AnselemOdims reviewed (changes requested):
### Hi @engrabdullahriaz , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

OmarRamoun reviewed (changes requested):
# 🟡 Just One More Thing 🔸

## Hi @engrabdullahriaz 👋

Great work on making the changes requested by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯, but **there are still some which aren't addressed yet**.

Though, There are some **few** issues 🐛 that you still need to work 🔧 on to go to the next project ⏭️ but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Required Changes ♻️

- [x] [REQUIRED] Please, implement the previous reviewer's requested changes (you didn't implement any)

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@mrramoun](https://img.shields.io/github/followers/MrRamoun?label=Ramoun&style=social)](https://github.com/mrramoun)
- This is My **Twitter**: [![@ramoun16](https://img.shields.io/twitter/follow/ramoun16?label=ramoun16&style=social)](https://twitter.com/ramoun16)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

engrabdullahriaz committed:
Updated html file and fixed issues

engrabdullahriaz committed:
Updated stylesheet and fixed styling of last sections

engrabdullahriaz committed:
updated stylesheet an fixed linter error

aimemalaika commented on index.html:
```diff
@@ -0,0 +1,136 @@
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+        </div>
+      </section>
+      <!--end of work section-->
+
+      <!--end about me section-->
+     <section class="about_me">
+        <div class="about_container">
+          <h3 class="about_me_title"> About <br>Myself</h3>
+          <div class="about_info">
+            <h3 class="about_me_title-desktop">About me</h3>
+            <div class="about_description">
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
```
- [ ] Kindly make sure the design in conforming the Figma mockup
- [ ] Kindly separate the see project button and the about section
<img width="435" alt="Screen Shot 2022-04-27 at 13 30 47" src="https://user-images.githubusercontent.com/28307960/165509158-5caf6f9e-e902-435c-8af2-24c23306a89f.png">
- [ ] Kindly add at least 4 projects

aimemalaika commented on style.css:
```diff
@@ -0,0 +1,415 @@
+.container-work {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto fit);
+  place-content: left;
```
- [x] Kindly fix these two lines as they are invalid property values

aimemalaika reviewed (changes requested):
Hi @engrabdullahriaz, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/28307960/157369993-50d5ba12-099f-4f2a-9a19-5e57ee2d1c96.gif)

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

engrabdullahriaz committed:
Added 4 projects and their images

engrabdullahriaz committed:
Updated html file

Added 4 projects and settelled about myself section and other sections too.

engrabdullahriaz committed:
Updated css file

updated styles for projects sections and about myself.

engrabdullahriaz committed:
Updated html file

Added projects and their images and removed linter error occured during changes.

engrabdullahriaz committed:
Updated css file

fixed linter error which occured during project changes,

roshan-bajgain commented on index.html:
```diff
@@ -0,0 +1,217 @@
+    </div>
+
+  </section> 
+      <!--work section-->
+      <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/SnapshootPortfolio0.png" alt="">
+        </div>
+        <div class="card-header">
+          <h2>Tonic</h2>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Back End Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+        </div>
+      </section>
+
```
- [x] I would suggest you make changes and stick as much as possible to Figma design.
- [x] kindly make the `li` tag looks similar to the Figma design and makes the less spacing between the `h2` tag and the `li` tag.
- [x] Kindly change the styling of the `li` tag (HTML, CSS, javascript) and make sure it looks like Figma design
- [x] kindly implement all the changes in all the section
### your design
![your-design](https://user-images.githubusercontent.com/81727988/165927885-7422cb9c-e84b-4d4c-8143-ff565f8d6993.png)
## Figma design
![figma](https://user-images.githubusercontent.com/81727988/165927919-86e807f1-afe0-4442-b43e-a11299d97fbd.png)

roshan-bajgain reviewed (changes requested):
Hi there, 

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet

<img src="https://media1.giphy.com/media/KG5KpLlr1UEPvwgkiM/giphy.gif"/>

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: 🍾 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

engrabdullahriaz committed:
Updated css file

made required changes to li tag

engrabdullahriaz committed:
Updated css file

fixed linter error occured during changes

aimemalaika reviewed (approved):
Hi @engrabdullahriaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7](https://user-images.githubusercontent.com/28307960/158747852-e9f4c957-d810-4de4-b468-69b1337714ea.png)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
